samples/bluetooth/peripheral_lbs/prj_minimal.conf

```diff
@@ -107,5 +107,6 @@ CONFIG_BT_BUF_EVT_RX_COUNT=2
 CONFIG_BT_CONN_TX_MAX=3
 CONFIG_BT_L2CAP_TX_BUF_COUNT=2
 CONFIG_BT_ATT_TX_COUNT=2
+CONFIG_BT_BUF_EVT_RX_COUNT=4
```

Comment:
where this nr comes from?

Reply:
thanks but that's just a CI test. No need for reviews here.